ets-ecole-de-technologie-superieure.csl

```diff
@@ -211,26 +210,41 @@
           <group delimiter=". " prefix=". ">
             <group>
               <text value="In "/>
-              <text macro="journal-title"/>
+              <choose>
+                <if variable="event">
+                  <text variable="event" font-style="italic"/>
+                </if>
+                <else>
+                  <text macro="journal-title"/>
+                </else>
+              </choose>
             </group>
             <group delimiter=", ">
-              <group delimiter=", " prefix="( " suffix=" )">
+              <group delimiter=", " prefix="(" suffix=")">
                 <text variable="event-place"/>
+								<text macro="date-range"/>
               </group>
               <text variable="page" prefix="p. "/>
             </group>
-            <text macro="publisher"/>
+            <text variable="publisher"/>
           </group>
         </else-if>
+				<else-if type="report" match="any">
+					<group delimiter=". " prefix=". ">
+						<group delimiter=", ">
+							<text macro="collection"/>
+							<text variable="number"/>
+						</group>
+						<group delimiter=", ">
+							<text macro="publisher"/>
+							<text variable="page" suffix=" p"/>
+						</group>
+					</group>
+				</else-if>
         <else-if type="webpage" match="any">
-          <group delimiter=". " prefix=". ">
-            <group>
-              <text value="In "/>
-              <text macro="journal-title"/>
-            </group>
-            <text value="En ligne"/>
-            <text variable="URL" prefix="&lt; " suffix="&gt;"/>
-            <text macro="accessed-date"/>
+          <group prefix=". ">
+						<text value="In "/>
```

Comment:
use text term, not value (and then set to text-case="capitalize-first" and put the space in the group delimiter).